src/components/material/material-buttons/MaterialButtons.tsx

```diff
@@ -25,20 +25,21 @@ const MaterialButtons: FC<MaterialButtonsProps> = ({
 
   return (
     <>
-      {hasCorrectAccessType(AccessTypeCode.Physical, manifestation) && (
-        <>
-          <MaterialButtonsPhysical
-            manifestation={manifestation}
-            size={size}
-            dataCy={`${dataCy}-physical`}
-          />
-          <MaterialButtonsFindOnShelf
-            size={size}
-            faustIds={[faustId]}
-            dataCy={`${dataCy}-find-on-shelf`}
-          />
-        </>
-      )}
+      {hasCorrectAccessType(AccessTypeCode.Physical, manifestation) &&
+        !isArticle(manifestation) === true && (
```

Comment:
Also consider adding an inline (or commit) comment explaining why we do not want to show these buttons for articles even though they are considered physical.

Note that you may have to restructure your code. We probably want to avoid having an more or less lengthy comment in the template.

Reply:
Done :) Thanks. As a part of this I found out that I misspelled something in the code too. 2 for the price of 1 😉 🛠️